Accessibility/VirtualContent/explainer.md

```diff
@@ -129,6 +129,72 @@ window.addEventListener("scroll", on_scroll_changed);
 13. Eventually, the backing server sends a content payload which also contains a signal that the end of the document has been reached. Script code removes the `aria-virtualcontent` attribute from the `main` element to indicate there is no more virtualized content.
 14. The next time the user attempts to navigate to the next heading, the AT searches for a virtual content container, finds none, and announces there are no further headings in the document.
 
+## Example Use 2: Tabular data
+
+This example illustrates one use of `aria-virtualcontent` for virtualized content in the inline direction. The document in this scenario is a table of bug reports with the following columns: report ID, report date, status, assigned to, title, fix date, fixed by. The last two columns have been virtualized out of the DOM based on the viewport width. When realized, these two columns do not have any data for bugs that are not yet fixed.
+
+```
+<html>
+<body>
+<table aria-virtualcontent="block-end">
+  <thead>
+    <tr aria-virtualcontent="inline-end">
+      <th>ID</th>
+      <th>Status</th>
+      <th>Assigned To</th>
+      <th>Report Date</th>
+      <th>Title</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>338</td>
+      <td>Assigned</td>
+      <td>Alice</td>
+      <td>April 21</td>
+      <td>Widget freezes up when the network is slow</td>
+    </tr>
+    <tr aria-virtualcontent="inline-end">
+      <td>342</td>
+      <td>Fixed</td>
+      <td>Bob</td>
+      <td>April 22</td>
+      <td>Crash when shift-double-clicking the widget</td>
+    </tr>
+    <tr>
+      <td>346</td>
+      <td>Untriaged</td>
+      <td>Unasssigned</td>
+      <td>April 25</td>
+      <td>Widget doesn't repaint when user rotates screen</td>
+    </tr>
+  </tbody>
+</table>
+</body>
+</html>
+```
+
+### Walkthrough
+
+With appropriate backing script, an AT navigating the above content by table cells might result in the following flow:
+1. The AT stops on and reads out each of the five realized cells in the header row.
+2. Upon reaching the end of the row, the AT discovers that the header row is a virtual content container and scrolls the document in the inline-end direction.
+3. Backing script realizes headers and data for the "Fix date" and "Fixed by" columns. It also virtualizes headers and data for the "ID" and "Status" columns. There are no longer any virtualized cells in the inline-end direction, but all rows in the table now have virtualized cells in the inline-start direction. Accordingly, script sets `aria-virtualcontent="inline-start"` on all rows.
```

Comment:
If this is the web developer's script, what triggers realization?

Reply:
Yes. Also, Script always refers to JavaScript running in the browser.